pkg/server/controller.go

```diff
@@ -256,6 +260,17 @@ func (ctrl *Controller) serverMux() (http.Handler, error) {
 		ctrl.drainMiddleware,
 		ctrl.authMiddleware(nil))
 
+	if ctrl.adminController != nil {
+		adminRouter := apiRouter.PathPrefix("/admin/").Subrouter()
+		adminRouter.Use(
+			api.AuthMiddleware(nil, ctrl.authService, ctrl.httpUtils),
+			authz.NewAuthorizer(ctrl.log, httputils.NewDefaultHelper(ctrl.log)).RequireOneOf(
+				authz.Role(model.AdminRole),
+			))
+		adminRouter.HandleFunc("/apps", ctrl.adminController.HandleGetApps).Methods("GET")
+		adminRouter.HandleFunc("/apps", ctrl.adminController.HandleDeleteApp).Methods("DELETE")
```

Comment:
Just a thought: shouldn't the route be `DELETE /apps/{app_name}`? I understand that this would require changing the handler, but it's still worth considering :)

Reply:
In https://github.com/pyroscope-io/pyroscope/issues/1263 a potential object in the future has sampleType field 
What if we would like to delete an app for only specific sampleType? Do we also put it in url? as path or query? Right now current DeleteInput json seems extendable the same way the new proposed returned object from `GET /apps`

Reply:
Good points. I'd say that we could represent an application as a collection of profile types: /apps/{app_name}/profile-types/{profile_type}

Let's maybe leave it as it is and revisit later